0.5.0

```diff
@@ -0,0 +1,40 @@
+Collecting package metadata (current_repodata.json): ...working... done

```

Comment:
Please remove this!

Reply:
I realize this is my fault because the developer guide offers bad guidance. In the new version it has been updated to add quotes around the version number to avoid this: ``conda install -c pyviz "pyctdev>0.5.0"``